frontend/src/metabase-lib/parameters/utils/click-behavior.ts

```diff
@@ -0,0 +1,330 @@
+import _ from "underscore";
```

Comment:
Unfortunately GitHub does not detect this as a moved file due to number of changes.
I left comments on lines that have changed (except added types, obviously).